### DISCUSSION TITLE
WASI-NN and Containers

### DISCUSSION THREAD

**Discussion by omkarprabhu-98** [WasmEdge/WasmEdge#2367] [Q&A]
Hi,

I'm trying to run the [Neutral network example](https://wasmedge.org/book/en/write_wasm/rust/wasinn.html#get-wasmedge-with-wasi-nn-plug-in-openvino-backend) for WASI-NN - OpenVINO backend but in a [container](https://wasmedge.org/book/en/use_cases/kubernetes/demo/wasi.html) (crun with wasmedge support) 

Is there any guidance on how to pass the mount arguments as shown below: 
```
wasmedge --dir .:. <wasm_file>.wasm mobilenet.xml mobilenet.bin input.jpg
```

in the Dockerfile for the container (standard one show below)
```
FROM scratch
ADD <wasm_file>.wasm /
CMD ["/<wasm_file>.wasm"]
```

Thanks!

**Comment by alabulei1** [comment]
For this question, please refer to [this issue](https://github.com/WasmEdge/WasmEdge/issues/2352). Feel free to give us some feedback. Thanks.

**Comment by omkarprabhu-98** [comment]
CRUN added support for this https://github.com/containers/crun/pull/1171
